libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -380,6 +380,8 @@ void RangeFinder::detect_instance(uint8_t instance, uint8_t& serial_instance)
         break;
     case RangeFinder_TYPE_LWI2C:
         if (params[instance].address) {
+            // the LW20 needs a long time to boot up, so we delay 1.5s here
+            hal.scheduler->delay(1500);
```

Comment:
Presumably this should be bypassed if it's a watchdog reboot? Just noting that we are gaining lots of ways to skip fast boots for the watchdog scenario.

Reply:
fixed, thanks!